src/bulkExport/getJobStatus.test.ts

```diff
@@ -6,9 +6,11 @@
 import * as AWSMock from 'aws-sdk-mock';
 import AWS from 'aws-sdk';
 import { QueryInput } from 'aws-sdk/clients/dynamodb';
+import sinon from 'sinon';
```

Comment:
nit: It's weird to use `sinon` instead of `jest.fn()` given that we are using jest. Mixing test libraries adds complexity to tests and jest alone is sufficient for these tests (I think).

I know other tests in this package use `sinon` but I'm not sure if that was needed to begin with.

No need to rewrite tests. Just something to keep in mind.

https://jestjs.io/docs/expect#tohavebeencalledwitharg1-arg2-
https://jestjs.io/docs/mock-function-api#mockfnmockcalls

Reply:
Yep, that makes sense. I'll try avoid using sinon going forward.